documentation/staging/content/userguide/managing-operators/preparation.md

```diff
@@ -54,29 +53,6 @@ Before installing an operator, ensure that each of these prerequisite requiremen
 
 1. Optionally, enable [Istio]({{< relref "/userguide/istio/istio.md" >}}).
 
-#### Download the operator source, if needed
-
-Downloading the operator source is required if:
-
-- You plan to provide local file-based access to the operator Helm chart
-  (see [Set up the operator Helm chart access](#set-up-the-operator-helm-chart-access)).
-- You need to manually install the CRD
-  (see [Manually install the Domain resource custom resource definition (CRD), if needed](#manually-install-the-domain-resource-custom-resource-definition-crd-if-needed)).
```

Comment:
CRD use case is going away, but file based Helm chart access is not.  File based Helm chart access should now have this material that was deleted (or a different way to get at the helm chart files).

Reply:
@tbarnes-us 1. Please note that I did **not** remove the "Manually install the Domain resource custom resource definition (CRD), if needed" section.
2. If, as you describe, the process changes, then the changes will be reflected in the docs.
3. Regarding the file-based Helm chart access, see the comments in the following section.

Reply:
@rosemarymarano I think both Ryan and I would prefer to find a way to not pollute the main flow of the docs that new users are reading with this obscure use case (i.e., the "Manually install the Domain resource custom resource definition (CRD), if needed" section).  Not sure how best to accomplish that.

Reply:
@rosemarymarano the "Manually install the Domain resource custom resource definition (CRD), if needed" section depends on the operator source today.  Regardless, this part of the doc will change significantly in 4.0 with Anil's upcoming updates...